pkg/limayaml/default.yaml

```diff
@@ -146,3 +102,67 @@ containerd:
 #    hint: |
 #      vim was not installed in the guest. Make sure the package system is working correctly.
 #      Also see "/var/log/cloud-init-output.log" in the guest.
+
+# ===================================================================== #
+# FURTHER ADVANCED CONFIGURATION
+# ===================================================================== #
+
+firmware:
+  # Use legacy BIOS instead of UEFI.
+  # Default: false
+  legacyBIOS: false
+
+video:
+  # QEMU display, e.g., "none", "cocoa", "sdl".
+  # As of QEMU v5.2, enabling this is known to have negative impact
+  # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
+  # Default: "none"
+  display: "none"
+
+network:
+  # The instance can get routable IP addresses from the vmnet framework using
+  # https://github.com/lima-vm/vde_vmnet. Both vde_switch and vde_vmnet
+  # daemons must be running before the instance is started. The interface type
+  # (host, shared, or bridged) is configured in vde_vmnet and not lima.
+  vde:
+    # url points to the vde_switch socket directory, optionally with vde:// prefix
+    # - url: "vde:///var/run/vde.ctl"
+    #   # VDE Switch port number (not TCP/UDP port number). Set to 65535 for PTP mode.
+    #   # Default: 0
+    #   port: 0
```

Comment:
Maybe call it `switchPort` to make it more explicit? Because people may delete the comments in their own config files and then later get confused.

We already use `macAddress` below, so we are not aiming for the shortest possible names.

Reply:
updated